index.js

```diff
@@ -24,11 +24,25 @@ function forEach(xs, fn) {
 }
 
 // TODO: use object-keys
-var objectKeys = Object.keys || function keys(obj) {
+function objectKeys(obj) {
 	var res = [];
-	for (var key in obj) { res.push(key); } // eslint-disable-line no-restricted-syntax
+
+	for (var key in obj) { // eslint-disable-line no-restricted-syntax
+		res.push(key);
+	}
+
+	if (Object.getOwnPropertySymbols) {
+		var symbols = Object.getOwnPropertySymbols(obj);
+
+		for (var i = 0; i < symbols.length; i++) {
+			if (Object.prototype.propertyIsEnumerable.call(obj, symbols[i])) {
```

Comment:
these builtin functions should be cached at module level

Reply:
Done. What is the reason for this? Is it to avoid any possible changes to these methods?

Reply:
Yep, exactly.